resources/views/Widgets/Grid/TabWidget.json

```diff
@@ -0,0 +1,33 @@
+{
+    "identifier": "Ceres::TabWidget",
+    "label": "Widget.tabLabel",
+    "previewImageURL": "/images/widgets/tab.svg",
+    "type": "structure",
+    "categories": ["structure"],
+    "position": 500,
+    "widgetClass": "Ceres\\Widgets\\Grid\\TabWidget",
+    "settings": {
+        "appearance": {
+            "type": "select",
+            "required": true,
+            "defaultValue": "primary",
+            "options": "@include(appearance)"
+        },
+        "tabs": {
+            "type": "vertical",
+            "isList": "[1,]",
+            "options": {
+                "name": "Tabs"
+            },
+            "children": {
+                "title": {
+                    "type": "text",
+                    "defaultValue": "Neues Tab"
```

Comment:
Da wir diesen Wert nicht in die Properties auslagern können, würde ich hier keinen defaultValue angeben. Dieser Fall müsste dann noch in der Tab-Komponente bzw. im Widget abgefangen werden.